resources/sass/app.scss

```diff
@@ -120,19 +120,18 @@ li {
 
     .auth-widget {
         display: flex;
-        color: $color-base-50;
```

Comment:
This property appears to be redundant here, as auth-widget links get their color from the link mixin.